### PR TITLE
rhombic & deltoidal

### LINKED ISSUE #39: clipping bug
In https://observablehq.com/@fil/oronce-fines-triangle-projection the graticule was properly clipped until d3-geo-polygon@1.7, but it failed with release [1.8.0](https://github.com/d3/d3-geo-polygon/releases/tag/v1.8.0).
I think the offending commit is  https://github.com/d3/d3-geo-polygon/commit/eb1e6635c35c3d21eb32c292115922205e89d153 ([#160](https://github.com/d3/d3-geo/issues/160)).

Test @ https://observablehq.com/@fil/clipping-bug-39

This is also the reason why #60 (rhombic) had a small rotation. Same bug appears in https://observablehq.com/d/8085f693f9972f0e (#9)

### LINKED ISSUE #57: deltoidal hexecontahedron projection
Hi,

i created a deltoidal hexecontahedron projection which can be seen here.

https://observablehq.com/d/881a8431e638b408

would this be a welcome addition to the project? I based it on d3.geoPolyhedralVoronoi()

### PR DESCRIPTION
**Pull request by Fil**
local copy of #59 ;

closes #57
closes #39

## deltoidal hexecontahedral
deltoidalHexecontahedral

## rhombic dodecahedral
rhombicDodecahedral

### CODE REVIEW

**Comment by Fil** [comment]
@rbathoorn this integrates your code, with a few tweaks:
* changed the base tooling to be more modern (we now have unit tests running in CI!)
* the new projections are listed in the README
* edited the default angle, scale, translate parameters so that the maps would fit in the usual 960x500 frame
* modified the default rotate
* tweaked the implementation to more modern JavaScript

File: src/rhombic.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    8, // 9
+    5, // 10
+    10, // 11
+  ];
+
+  return voronoi()
+    .polygons(polygons)
+    .parents(parents)
+    .angle(20)
+    .rotate([0.001, 0])
```
Comment (Fil) [inline src/rhombic.js:85]: see #39

**Comment by Fil** [comment]
This feels like a better default aspect:

`d3.geoRhombicDodecahedral().rotate([80, 0, -Math.asin(Math.sqrt(3) / 3) * 90])`

rhombic

**Comment by bathoorn** [comment]
> This feels like a better default aspect:
> 
> `d3.geoRhombicDodecahedral().rotate([80, 0, -Math.asin(Math.sqrt(3) / 3) * 90])`
> 
 Look better indeed. And the names of the functions could be shortened to something like geoRhombic and geoDeltoidal I just put in the whole names as all the previous ones also had the complete names :)

**Pull request merged**